common/src/main/java/com/wynntils/models/items/items/gui/TerritoryItem.java

```diff
@@ -127,4 +131,11 @@ public String toString() {
                 + alerts + ", upgrades="
                 + upgrades + '}';
     }
+
+    public static class TerritorySerializer implements JsonSerializer<TerritoryItem> {
+        @Override
+        public JsonElement serialize(TerritoryItem territoryItem, Type type, JsonSerializationContext context) {
+            return null;
+        }
+    }
```

Comment:
Left behind?

Reply:
Yeah, IntelliJ and Git don't seem to like each other